js/src/carousel.js

```diff
@@ -418,7 +418,7 @@ class Carousel extends BaseComponent {
       return
     }
 
-    const elementInterval = Number.parseInt(element.getAttribute('data-bs-interval'), 10)
+    const elementInterval = Manipulator.getDataAttribute(element, 'interval')
```

Comment:
`Number.parentInt` should probably stay, right?

Reply:
The data is already being normalized in manipular:
```js
if (val === Number(val).toString()) {
  return Number(val)
}
```

But that does not parse the number in integer though :thinking:

Reply:
Updated :+1: